lessons/pair-programming/README.md

```diff
@@ -0,0 +1,39 @@
+- title: Intro to Pair Programming
+- tags: pair programming
+
+# Objectives
+* Be able to pair program effectively
+
+# Resources
+- [Intro to Pair Programming Concepts](http://cs.brown.edu/courses/csci0170/content/docs/pair-programming.pdf)
+- [Pair Programming Tips](http://www.wikihow.com/Pair-Program)
+- [How to Pair Program](https://medium.com/@dickeyxxx/how-to-pair-program-d6741077e513#.wsqzkwyeq)
+
+#Lecture
+##Intro to pairing
+Pair programming is a great learning technique that you'll encounter again and again throughout your coding journey. 
+
+Pairing is basically the idea of two programmers working together on a single computer. Pairing helps you get better at articulating your thoughts and sharing your ideas with your peers, which is a crucial skill for any programmer. It will also help you catch your errors and ultimately help you produce better, bug-free code, which is why it is used at companies throughout the tech industry.
+
+##Why pair program?
+* You will produce better code 
+* You will learn more, sharing your ideas with your peers and benefiting from their insights 
+* You will become better at articulating your thoughts
+* You will enjoy your work more and spend less time frustrated
+* You will be better prepared for more complicated software engineering tasks, both in school
+and beyond, where collaboration is imperative to success
+
+(Sourced from [Brown's CS 17 intro to pairing](http://cs.brown.edu/courses/csci0170/content/docs/pair-programming.pdf))
+
+##How to pair
+While pairing, you should have the following two roles:
+1. **The Driver:** The driver should type and physically control the computer
+2. **The Navigator:** The navigator should instruct the driver on the overall, high-level strategy and direction to take the code. The navigator should also be looking out for mistakes, and should inform the driver.
+
+Be sure to **switch roles every 15-30 minutes**! 
+
+Other tips:
+- Talk a lot! Try to talk things out as much as possible
+- Pay close attention to the code and try to catch errors as early as possible
+- Stick to your roles until the agreed upon switching point; navigator should not be grabbing the keyboard/typing when it is not their role, and vice versa
+- Rely on your partner and support your partner. Work together as a team! 
```

Comment:
Love all these tips! Important to state explicitly, glad you're putting them down for students.